alert-database/src/main/java/com/synopsys/integration/alert/database/api/StaticJobAccessor.java

```diff
@@ -0,0 +1,276 @@
+    }
+
+    @Override
+    @Transactional(readOnly = true)
+    public AlertPagedModel<ConfigurationJobModel> getPageOfJobs(int pageOffset, int pageLimit, Collection<String> descriptorsNamesToInclude) {
+        if (!descriptorsNamesToInclude.contains(blackDuckProviderKey.getUniversalKey())) {
+            return new AlertPagedModel<>(0, pageOffset, pageLimit, List.of());
+        }
+
+        PageRequest pageRequest = PageRequest.of(pageOffset, pageLimit);
+        Page<ConfigurationJobModel> pageOfJobsWithDescriptorNames = distributionJobRepository.findByChannelDescriptorNameIn(descriptorsNamesToInclude, pageRequest)
+                                                                        .map(this::convertToConfigurationJobModel);
+        return new AlertPagedModel<>(pageOfJobsWithDescriptorNames.getTotalPages(), pageOffset, pageLimit, pageOfJobsWithDescriptorNames.getContent());
+    }
+
+    @Override
+    @Transactional(readOnly = true)
+    public Optional<ConfigurationJobModel> getJobById(UUID jobId) {
+        return distributionJobRepository.findById(jobId).map(this::convertToConfigurationJobModel);
+    }
+
+    @Override
+    @Transactional(readOnly = true)
+    public Optional<ConfigurationJobModel> getJobByName(String jobName) {
+        return distributionJobRepository.findByName(jobName)
+                   .map(this::convertToConfigurationJobModel);
+    }
+
+    @Override
+    @Transactional(readOnly = true)
+    public List<ConfigurationJobModel> getJobsByFrequency(FrequencyType frequency) {
+        return distributionJobRepository.findByDistributionFrequency(frequency.name())
+                   .stream()
+                   .map(this::convertToConfigurationJobModel)
+                   .collect(Collectors.toList());
+    }
+
+    @Override
+    @Transactional
+    public ConfigurationJobModel createJob(Collection<String> descriptorNames, Collection<ConfigurationFieldModel> configuredFields) {
+        return createJobWithId(null, configuredFields, DateUtils.createCurrentDateTimestamp(), null);
+    }
+
+    @Override
+    @Transactional
+    public ConfigurationJobModel updateJob(UUID jobId, Collection<String> descriptorNames, Collection<ConfigurationFieldModel> configuredFields) throws AlertDatabaseConstraintException {
+        DistributionJobEntity jobEntity = distributionJobRepository.findById(jobId)
+                                              .orElseThrow(() -> new AlertDatabaseConstraintException(String.format("No job exists with the id [%s]", jobId.toString())));
+        OffsetDateTime createdAt = jobEntity.getCreatedAt();
+
+        deleteJob(jobId);
+        return createJobWithId(jobId, configuredFields, createdAt, DateUtils.createCurrentDateTimestamp());
+    }
+
+    @Override
+    @Transactional
+    public void deleteJob(UUID jobId) {
+        distributionJobRepository.deleteById(jobId);
+    }
+
+    private ConfigurationJobModel convertToConfigurationJobModel(DistributionJobEntity jobEntity) {
```

Comment:
Should this return DistributionJobModel instead of ConfigurationJobModel at this point? Same for the methods above?

Reply:
This PR was about reproducing an identical API with the new tables. The intention is to move to the new models eventually, but it is out of scope for this PR as it would require changes to dozens of other files.

Reply:
I am on board with that if you add a TODO to change the return type to DistributionJobModel in the future